test/CMakeLists.txt

```diff
@@ -59,24 +60,24 @@ pmpo_add_fortran_exe(testFortranCreateRebuildMPs testFortranCreateRebuildMPs.f90
 #add test
 find_program(MPIRUN_EXECUTABLE NAMES mpirun) 
 
-function(pmpo_add_test testname command)
+function(pmpo_add_test testname procs command)
```

Comment:
Is MPI required in PolyMPO?  I can't remember why we have the `if(MPIRUN_EXECUTABLE)` conditional.

Reply:
I don't see anything requiring MPI